### PR TITLE
draft readme.md file

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Remove the .rst readme as it is not compatible with Galaxy. Replaced with a more detailed README.md file.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

documentation
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by danielmellado** [comment]
recheck

**Comment by samccann** [comment]
recheck

**Comment by samccann** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by samccann** [comment]
recheck

**Comment by danielmellado** [comment]
Let's see how does this go on this roll, if it gets stuck again we'll have to take a deeper look. Thanks!

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
ok, I've set up an autohold there, let's debug on the node when it's ready

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by trishnaguha** [comment]
@samccann @danielmellado @rohitthakur2590 @pabelanger Is the blocker/CI failure for getting this PR merged being fixed yet? Are we good for another recheck?

**Comment by samccann** [comment]
FWIW the other readme for netcommon is also failing CI.

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+- [Ansible Collection overview](https://github.com/ansible-collections/overview)
+- [Ansible User guide](https://docs.ansible.com/ansible/latest/user_guide/index.html)
+- [Ansible Developer guide](https://docs.ansible.com/ansible/latest/dev_guide/index.html)
+- [Ansible Community code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html)
+
+## Licensing
+
+GNU General Public License v3.0 or later.
+
+See [LICENCE](https://www.gnu.org/licenses/gpl-3.0.txt) to see the full text.
```
Comment (samccann) [inline README.md:147]:
```suggestion
See [LICENSE](https://www.gnu.org/licenses/gpl-3.0.txt) to see the full text.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+### See Also:
+
+* [VyOS Platform Options](https://docs.ansible.com/ansible/latest/network/user_guide/platform_vyos.html)
+* [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.
+
+## Contributing to this collection
+
+We welcome community contributions to this collection. If you find problems, please open an issue or create a PR against the [VyOS collection repository](https://github.com/ansible-collections/vyos).
+
+You cal also join us on:
```
Comment (samccann) [inline README.md:120]:
```suggestion
You can also join us on:
```

**Comment by trishnaguha** [comment]
@samccann The CI is passing now, will you please resolve the conflict? Thanks!

File: README.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+You can either call modules by their Fully Qualified Collection Namespace (FQCN), like `vyos.vyos.vyos_l3_interfaces`, or you can call modules by their short name if you list the `vyos.vyos` collection in the playbook's `collections`, as follows:
+
+```yaml
+---
+- hosts: vyos01
+  gather_facts: false
+  connection: network_cli
+
+  collections:
```
Comment (ganeshrn) [inline README.md:47]: collections keyword is not required if FQCN is used in task name. Probably can be split using collections keyword and only task name and other without collection name and FQCN task name.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+                     - forward_router_address: 192.0.2.9
+       state: replaced
+```
+
+
+
+### See Also:
+
+* [VyOS Platform Options](https://docs.ansible.com/ansible/latest/network/user_guide/platform_vyos.html)
```
Comment (ganeshrn) [inline README.md:113]: The platform guide needs to be updated to use FQCN name as well for example change `ansible_connection: network_cli` to `ansible_connection: ansible.netcommon.network_cli`

File: README.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+* [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.
+
+## Contributing to this collection
+
+We welcome community contributions to this collection. If you find problems, please open an issue or create a PR against the [VyOS collection repository](https://github.com/ansible-collections/vyos).
```
Comment (ganeshrn) [inline README.md:118]: We can add a pointer to contributor guide here whenever it is public.

File: README.md (hunk 4 above)
Comment (samccann) [reply README.md:116]: the internal spreadsheet is tracking those changes to platform docs now.

**Comment by trishnaguha** [comment]
@samccann Can we merge this PR now?

**Comment by samccann** [comment]
@trishnaguha yes it's ready to merge!

**Pull request merged**